docs/services/navigation/_index.md

```diff
@@ -115,6 +119,54 @@ The following attributes are available for `Navigation` services:
 | `meters_per_sec` | float | Optional | The default linear velocity for the [base](/components/base/) in meters per second. <br> Default: `0.3` |
 | `obstacles` | obj | Optional | Any obstacles you wish to add to the robot's path. See the [Motion Service](/services/motion/) for more information. |
 
+### Configure and calibrate the Frame System Service for GPS Navigation
+
+{{% alert title="Info" color="info" %}}
+
+The [Frame System Service](/services/frame-system/) is an internally managed and mostly static system for storing the "reference frame" of each component of a robot within a coordinate system configured by the user.
```

Comment:
```suggestion
The [Frame System Service](/services/frame-system/) is an internally managed and mostly static system for storing the reference frame of each component of a robot within a coordinate system configured by the user.
```
I agree with Tess that this alert is probably not necessary and can come out in favor of just linking to the frame system doc. However, if it stays in, recommend taking the quote marks off here.